src/pages/pattern-library.tsx

```diff
@@ -66,6 +66,18 @@ function Page() {
               </li>
             </ul>
           </section>
+
+          <section className="grid-section grid-content">
+            <h2 className="title-subsection">Badge</h2>
+            <ul className="list-horizontal">
+              <li>
+                <UIBadge>15% off</UIBadge>
+              </li>
+              <li>
+                <UIBadge className="badge_small">15% off</UIBadge>
+              </li>
+            </ul>
+          </section>
```

Comment:
Something is broken here:

<img width="1427" alt="image" src="https://user-images.githubusercontent.com/8497187/146778283-a2788275-8a65-491a-b77f-18acdd2b8f91.png">

Reply:
I'll take a look into!

Reply:
Fixed! 🤩